java_v1/README.md

```diff
@@ -0,0 +1,9 @@
+Java
+
+* Simple java implementation (single threaded)
+* Docker noob here - I will try to come up with a Dockerfile later.
+* To run this program:
+	- javac App.java
+	- java App
+	- If you are behind a proxy: java -Dhttps.proxyHost=<PROXY_HOST> -Dhttps.proxyPort=<PROXY_PORT> App
+
```

Comment:
I didn't know this trick.. I've not worked with Java for quite some time. This is very useful 👍